xml/dm_registers.xml

```diff
@@ -37,6 +36,12 @@
 
             7 (other): An access failed for another reason.
         </field>
+	<field name="allhasreset" bits="19" access="R" reset="-">
+          This field is 1 when all currently selected harts have been reset since the last write to its \Fhasresetack.
```

Comment:
... since the last time reset was acknowledged for them.

Referring to the last \Fhasresetack write seems confusing regarding which harts were/are selected.

Reply:
yeah, I agree it was awkward as I was writing it. I think this parallels the same things you are trying to address with #158